### DISCUSSION TITLE
What are the conditions for the word cloud?

### DISCUSSION THREAD

**Discussion by FabianK-Dev** [jendrikseipp/rednotebook#586] [Q&A] [answered]
I have some words which are under five characters long which I mentioned many times in RedNotebook but don't appear in the word cloud. Are there specific conditions for words to be able to show up in the cloud tag such as a minimum amount of characters?

Thanks for any help.

**Comment by jendrikseipp** [comment, marked as answer]
Yes, words have to have a minimum length. Apparently it's 5 characters.

**Reply by FabianK-Dev** [reply]
Thanks! :+1:
